### DISCUSSION TITLE
Can I make a PDF file of the packet information collected from SOC?

### DISCUSSION THREAD

**Discussion by kkdbudglf** [Security-Onion-Solutions/securityonion#10090] [Unsupported Versions]
(Please be informed that this article used a translator.)

I want to make the collected packet information into a PDF file that is easy to see at once.

Does this Security Onion have that?

**Comment by InfosecGoon** [comment]
No, there is no native PDF export function in SOC. Your best bet might be to create a Dashboard with the information you want and then take a screenshot or print as PDF.
